Extensions/PrimitiveDrawing/shapepainterruntimeobject-pixi-renderer.ts

```diff
@@ -485,8 +486,35 @@ namespace gdjs {
       point[1] = position.y;
       return point;
     }
+    applyAntialiasing(): void {
+      if (this._object.isAntialiasingOn()) {
+        let antialiasingFilter = new PIXI.filters.FXAAFilter();
```

Comment:
Instead of recreating one every time, you should cache the first instance you create, as to not recreate it when simply changing quality setting. You also want to use `const` instead of `let` whenever possible. Add a class property `antialiasingFilter: null | PIXI.filters.FXAAFilter = null;`, then change this line to:
```suggestion
        const antialiasingFilter = this.antialiasingFilter || (antialiasingFilter = new PIXI.filters.FXAAFilter());
```